src/builtins/core/date.rs

```diff
@@ -724,9 +725,11 @@ impl FromStr for PlainDate {
     }
 }
 
-#[cfg(test)]
+#[cfg(all(test, feature = "tzdb"))]
```

Comment:
nit: the feature configuration shouldn't be here I think. You should be able to configure specific tests. With `#[cfg(feature = "tzdb"]` above the specific tests. I think references can be found in other test modules.